addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java

```diff
@@ -82,6 +95,7 @@
     private final Logger logger = LoggerFactory.getLogger(EchoHandler.class);
     private Gson gson = new Gson();
     private @Nullable Device device;
+    private Set<String> capabilities = new HashSet<String>();
```

Comment:
Please use the diamond operator.

```suggestion
    private Set<String> capabilities = new HashSet<>();
```